deliverable/1/peer_review.md

```diff
@@ -0,0 +1,12 @@
+# Peer Code Review Checklist
+
+- [ ] Passes CI builds at least as well as before change
+- [ ] Task description satisfied accurately and in entirety
+- [ ] Sufficient tests for new or previously broken functionality
+- [ ] Formatted according to PEP8
+- [ ] No obvious bugs or oversight
+- [ ] Meaningful variable names
+- [ ] Any new dependencies documented and added to Dockerfile
+- [ ] No new pylint errors or warnings (unless justified and approved)
+- [ ] External Documentation (docstrings) for all new functions, classes. modules and methods
```

Comment:
`classes.` -> `classes,`

Reply:
also `modules and methods` -> `modules, and methods`

Reply:
Fixed grammar issue.